frontend/src/metabase/public/components/EmbedFrame/EmbedFrame.styled.tsx

```diff
@@ -86,6 +86,11 @@ const footerVariantStyles = {
   `,
 };
 
+export const ParametersWidgetContainer = styled.div`
+  display: flex;
+  align-items: baseline;
+`;
```

Comment:
Matches the apply button styles on normal app dashboards. See `frontend/src/metabase/dashboard/components/Dashboard/Dashboard.styled.jsx`